workers/run_command.sh

```diff
@@ -45,13 +48,13 @@ if [ ! -d "$volume_directory" ]; then
     chmod -R a+rwX "$volume_directory"
 fi
 
-source common.sh
+. ./scripts/common.sh
 HOST_IP=$(get_ip_address)
 DB_HOST_IP=$(get_docker_db_ip_address)
 
 chmod -R a+rwX "$volume_directory"
 
-./prepare_image.sh -i "$image" -s workers
+./scripts/prepare_image.sh -i "$image" -s workers
```

Comment:
I know it's not part of this PR but it looks like I accidentally committed some specific command stuff in the bottom of this script. Could you change it so that instead of running that `Rscript` command it just runs whatever was passed in as arguments to the script?